docs/user/glossary.rst

```diff
@@ -44,6 +44,19 @@ so that you have a reference for how we're using them.
       and rules for publication of documentation can be :doc:`automated </automation-rules>`.
       Similar to :term:`Docs as Code`.
 
+   maintainer
+      A *maintainer* is a special role that only exists on |org_brand|.
+      The creator of a project on |org_brand| can invite other collaborators as *maintainers* with full ownership rights.
+
+      At least one maintainer of a Read the Docs project should have admin rights to the Git repository.
+      This ensures that all settings of the Git repository can remain updated over time.
+
+      Git provider integration is active through the authentication of the maintainter that creates the integration.
+      If a maintainer is removed,
+      make sure to verify and potentially recreate all Git integration *if* removing the maintainer that originally configured this.
```

Comment:
I know that I'm bending the "normal" idea of definition here, but I think this information is associated with maintainers? I couldn't find it in the article you referenced, but if the info is there somewhere, I can replace the glossary part with a reference :+1: 

(I had a recent support inquiry that took a long time to resolve exactly because of this)

Reply:
Ah you wrote _probably_ duplicated :) I'm a bit scared of mentioning maintainers in that other article, although I know it's a good addition. It just "drags" in the need to also mention Organizations and teams and OAuth ownership.

Reply:
This information won't be found by users in our glossary, so even if that guide is not ideal placement, it's still *better*. I'd just dial back on things, we don't need to get into teams/orgs to make this small step. We can always come back and add more later.

Reply:
I've moved the text from the Glossary into a separate section. I hope this works. But I think you should just have a quick look again :blush: